test/test.js

```diff
@@ -0,0 +1,38 @@
+const puppeteer = require('puppeteer');
+const tap = require('tap');
+const pa11y = require('pa11y');
+
+async function test(server){
+  //General browser tests
+  await tap.test('The map is loaded', async function(t) {
+    const browser = await puppeteer.launch({args: ['--no-sandbox', '--disable-setuid-sandbox']});
+    const page = await browser.newPage();
+    await page.goto('http://localhost:8123/index.html');
+    const div = await page.$eval('#mapdiv', el => el ? true : false);
+    t.assert(div === true, 'div is present');
+    const mapCenter = await page.evaluate(() => map.getCenter());
+    t.assert(mapCenter.lng === 5.9699 && mapCenter.lat=== 52.2112, 'map is centered as expected')
+    await browser.close();
+    t.end()
+  })
+  //ARIA tests
+  await tap.test('there are no accessibility issues', async function(t) {
+    const ariares = await pa11y('http://localhost:8123/index.html', {
+      allowedStandards: 'WCAG2AA',
```

Comment:
Aria config in a separate file somewhere?

Reply:
Good idea. I see from https://github.com/Amsterdam/atlas/blob/develop/test/pa11y.js that my pa11y approach is a  bit naive still. I suppose we can copy the general setup from there?

Reply:
Yes. Do use that!